src/sensors.cpp

```diff
@@ -462,17 +459,19 @@ void Sensors::calibrate_baro()
       // else reset cal variables and start over
       if (baro_calibration_var_ < BARO_MAX_CALIBRATION_VARIANCE) {
         rf_.params_.set_param_float(PARAM_BARO_BIAS, baro_calibration_mean_);
+        // set ground altitude to be pressure altitude at PARAM_BARO_BIAS
+        rf_.params_.set_param_float(PARAM_GROUND_LEVEL,turbomath::alt(rf_.params_.get_param_float(PARAM_BARO_BIAS)));
         baro_calibrated_ = true;
-        rf_.comm_manager_.log(CommLinkInterface::LogSeverity::LOG_INFO, "Baro Cal successful!");
+        rf_.comm_manager_.log(CommLinkInterface::LogSeverity::LOG_INFO, "Baro ground pressure cal successful!");
       } else {
         rf_.comm_manager_.log(CommLinkInterface::LogSeverity::LOG_ERROR,
-                              "Too much movement for barometer cal");
+                              "Too much movement for barometer ground pressure cal");
       }
       baro_calibration_mean_ = 0.0f;
       baro_calibration_var_ = 0.0f;
       baro_calibration_count_ = 0;
     } else if (baro_calibration_count_ > SENSOR_CAL_DELAY_CYCLES) {
-      float measurement = data_.baro_pressure - ground_pressure_;
+      float measurement = data_.baro_pressure;
```

Comment:
If we don't use the ground pressure, is there a reason to even run the ground pressure calibration? Or am I missing something?

Reply:
The bias (ground pressure) is computed to provide a ground altitude/pressure reference for cases where other references are not available. It is accessible to the companion computer by reading the PARAM_BARO_BIAS register. For convenience the standard atmosphere model is used to also compute the PARAM_GROUND_LEVEL parameter.